src/main/java/com/igot/cb/transactional/cassandrautils/CassandraOperationImpl.java

```diff
@@ -201,6 +202,24 @@ public Map<String,Object> updateRecord(
         return response;
     }
 
+    @Override
+    public void deleteRecord(String keyspaceName, String tableName, Map<String, Object> compositeKeyMap) {
```

Comment:
there should not be any delete operation in cassandra... 
we should do only soft delete -- that is updating a column in that table mentioning record is inactive.

Reply:
Implemented soft delete